### PR TITLE
Update the game over message

### PR DESCRIPTION
**Pull request by Nadika18**
Update the game over message so people know how to play again!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,51 +14,30 @@ _Collaborate and work together on GitHub._
 </header>
 
 <!--
-  <<< Author notes: Course start >>>
-  Include start button, a note about Actions minutes,
-  and tell the learner why they should take the course.
+  <<< Author notes: Step 4 >>>
+  Start this step by acknowledging the previous step.
```
Comment (Nadika18) [inline README.md:18]: Complete above step and follow next.

File: README.md [hunk 2]
```diff
@@ -14,51 +14,23 @@ _Collaborate and work together on GitHub._
...
 <!--
-  <<< Author notes: Course start >>>
-  Include start button, a note about Actions minutes,
-  and tell the learner why they should take the course.
+  <<< Author notes: Step 5 >>>
+  Start this step by acknowledging the previous step.
+  Define terms and link to docs.github.com.
 -->
 
-## Welcome
```
Comment (Nadika18) [inline README.md:22]: Welcome to technicall onboarding.

File: README.md [hunk 3]
```diff
@@ -14,51 +14,23 @@ _Collaborate and work together on GitHub._
...
+_Nicely done suggesting changes! :partying_face:_
 
-- **Who is this for**: Developers, new GitHub users, users new to Git, students, managers, teams.
-- **What you'll learn**: When and how to request a review; how to provide a review of someone else's pull request.
-- **What you'll build**: We'll be reviewing a pull request for a simple game.
-- **Prerequisites**: We assume you are familiar with creating branches, commits, and pull requests—you can learn this in our [Introduction to GitHub](https://github.com/skills/introduction-to-github) course.
-- **How long**: This course takes less than 30 minutes to complete.
+Now let's see how easy it is to [apply your suggestion](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/incorporating-feedback-in-your-pull-request).
 
-In this course, you will:
```
Comment (Nadika18) [inline README.md:32]: Here

File: README.md (hunk 2 above)
Comment (krishbin) [inline README.md:22]: Welcome to the course

Comment (Nadika18) [reply README.md:22]: okay

File: index.html [hunk 4]
```diff
@@ -47,7 +47,7 @@ <h1>Tic Tac Toe</h1>
   </tr>
 </table>
 
-<h2 hidden>Game over</h2>
+<h2 hidden>Game over, refresh to play again 🧑‍💻 🤖!</h2>
```
Comment (krishbin) [inline index.html:51]: <h2>Hi</h2>

**Pull request merged**